packages/toolkit/scripts/project/init.js

```diff
@@ -177,6 +180,29 @@ const run = async () => {
 		});
 
 		files.forEach((file) => {
+			// Skip directories
+			if (fs.statSync(file).isDirectory()) {
+				return;
+			}
+
+			// Only process specific file extensions
+			const ext = path.extname(file).toLowerCase();
+			const allowedExtensions = [
+				'.ts',
+				'.js',
+				'.tsx',
+				'.jsx',
+				'.json',
+				'.php',
+				'.md',
+				'.yml',
+				'.yaml',
```

Comment:
```suggestion
				'.yaml',
				'.mustache',
				'.sh',
				'.xml',
				'.html',
				'.svg'
```

Should we maybe add a disallowed list instead?

Reply:
This allowlist is for files that would need search and replace, would t hese ones you added be required for search and replace?

Reply:
Yes they contain the theme name etc...